src/CDSAPI.jl

```diff
@@ -1,88 +1,103 @@
 module CDSAPI
 
+using Base64
 using HTTP
 using JSON
-using Base64
+using Logging
 
 export
-    py2ju
+        retrieve, cdskeys
 
 """
-    retrieve(name, params, filename)
-
-Retrieves data for `name` from the Climate Data Store
-with the specified `params` and stores it in the current
-directory as `filename`.
+    retrieve(
+        fname::AbstractString,
+        cdsdataset::AbstractString,
```

Comment:
I find this order confusing compared to the Python equivalent. Can we try to preserve the first argument as the name of the dataset in CDS, and the last argument as the name of the file saved?

Reply:
Alright, I resorted the arguments in the newest push!